aviary/visualization/dashboard.py

```diff
@@ -436,8 +455,8 @@ def dashboard(script_name, problem_recorder, driver_recorder, port):
         Name of the script file whose results will be displayed by this dashboard.
     problem_recorder : str
         Name of the recorder file containing the Problem cases.
-    driver_recorder : str
-        Name of the recorder file containing the Driver cases.
+    driver_recorder : str or None
+        Name of the recorder file containing the Driver cases. If None, ignore this file
```

Comment:
I'd prefer something more descriptive than "ignored", maybe something like:
```suggestion
        Name of the recorder file containing the Driver cases. If None, the driver tab will not be added
```

Reply:
Good suggestion! Fixed